tests/materials/test_concrete.py

```diff
@@ -0,0 +1,123 @@
+        """Tests the eps_cu1."""
+        assert fixture_concrete_material_c30_37.eps_cu1 == 3.5
+
+    def test_eps_cu1_high_concrete(self, fixture_concrete_material_c90_105: ConcreteMaterial) -> None:
+        """Tests the eps_cu1."""
+        assert fixture_concrete_material_c90_105.eps_cu1 == 2.8
+
+    def test_eps_c2_low_concrete(self, fixture_concrete_material_c30_37: ConcreteMaterial) -> None:
+        """Tests the eps_c2."""
+        assert fixture_concrete_material_c30_37.eps_c2 == 2.0
+
+    def test_eps_c2_high_concrete(self, fixture_concrete_material_c90_105: ConcreteMaterial) -> None:
+        """Tests the eps_c2."""
+        assert fixture_concrete_material_c90_105.eps_c2 == 2.6004968327615767
+
+    def test_eps_cu2_low_concrete(self, fixture_concrete_material_c30_37: ConcreteMaterial) -> None:
+        """Tests the eps_cu2."""
+        assert fixture_concrete_material_c30_37.eps_cu2 == 3.5
+
+    def test_eps_cu2_high_concrete(self, fixture_concrete_material_c90_105: ConcreteMaterial) -> None:
+        """Tests the eps_cu2."""
+        assert fixture_concrete_material_c90_105.eps_cu2 == 2.6
+
+    def test_n_factor_low_concrete(self, fixture_concrete_material_c30_37: ConcreteMaterial) -> None:
+        """Tests the n_factor."""
+        assert fixture_concrete_material_c30_37.n_factor == 2.0
+
+    def test_n_factor_high_concrete(self, fixture_concrete_material_c90_105: ConcreteMaterial) -> None:
+        """Tests the n_factor."""
+        assert fixture_concrete_material_c90_105.n_factor == 1.4
+
+    def test_eps_c3_low_concrete(self, fixture_concrete_material_c30_37: ConcreteMaterial) -> None:
+        """Tests the eps_c3."""
+        assert fixture_concrete_material_c30_37.eps_c3 == 1.75
+
+    def test_eps_c3_high_concrete(self, fixture_concrete_material_c90_105: ConcreteMaterial) -> None:
+        """Tests the eps_c3."""
+        assert fixture_concrete_material_c90_105.eps_c3 == 2.3
+
+    def test_eps_cu3_low_concrete(self, fixture_concrete_material_c30_37: ConcreteMaterial) -> None:
+        """Tests the eps_cu3."""
+        assert fixture_concrete_material_c30_37.eps_cu3 == 3.5
+
+    def test_eps_cu3_high_concrete(self, fixture_concrete_material_c90_105: ConcreteMaterial) -> None:
+        """Tests the eps_cu3."""
+        assert fixture_concrete_material_c90_105.eps_cu3 == 2.6
+
+    def test_rho_min(self, fixture_concrete_material_c30_37: ConcreteMaterial) -> None:
+        """Tests the rho_min."""
+        assert fixture_concrete_material_c30_37.rho_min(f_yd=250) == 0.2583649593204665
+
+    def test_eq(self, fixture_concrete_material_c30_37: ConcreteMaterial) -> None:
+        """Test equality checks."""
+        assert fixture_concrete_material_c30_37 == ConcreteMaterial(
+            concrete_class=ConcreteStrengthClass.C30_37,
+            name="different name doesn't affect equality",
+        )
+        assert fixture_concrete_material_c30_37 != ConcreteMaterial(concrete_class=ConcreteStrengthClass.C12_15)
```

Comment:
actuually you are testing 'inequality' (inverse of equality) instead of equality here.